app/src/main/java/ar/com/wolox/android/cookbook/animatedinput/CurrencyInputDouble.kt

```diff
@@ -0,0 +1,75 @@
+package ar.com.wolox.android.cookbook.animatedinput
+
+import android.content.Context
+import android.content.res.TypedArray
+import android.text.Editable
+import android.text.TextWatcher
+import android.util.AttributeSet
+import android.view.LayoutInflater
+import android.view.animation.Animation
+import android.view.animation.AnimationUtils
+import androidx.constraintlayout.widget.ConstraintLayout
+import com.example.inputtypes.R
+import com.example.inputtypes.databinding.CurrencyInputDoubleBinding
+
+/*
+*  Este componente es un ejemplo de como se podria implementar una animacion dentro de un textedit.
+*  El problema de animar el contenido de EditText es que no es un componente en si.
+*  El workaround que se propone es solapar un TextView sobre un EditText con texto transparente.
+*  Sobre este componente se pueden implementar muchas mas features (pasarle por atributos, por ejemplo
+*  el color del selector de texto, o el underline). Lo deje lo mas minimal posible para que se pueda
+*  extender o utilizar a gusto. 
+*/
+
+class CurrencyInputDouble @JvmOverloads constructor(
```

Comment:
Silly question, but why is it called Double?

Reply:
Because it admits a double as an input 😅. Thinking of, maybe I could change it to CurrencyInputDecimal

Reply:
I was thinking something like AnimatedEditText! What do you think?